proto/Networking/Requests/Messages/PlatformClientActionsMessage.proto

```diff
@@ -33,29 +34,68 @@ message OptOutPushNotificationCategoryMessage {
 	repeated string categories = 1;
 }
 
-message RedeemPasscodeMessage {
```

Comment:
was correct, request type is REDEEM_PASSCODE

Reply:
https://github.com/Furtif/POGOProtos/blob/master/src/POGOProtos/Enums/PlatformClientAction.proto#L12

Reply:
exactly, if request type is REDEEM_PASSCODE, then message is `RedeemPasscodeMessage`, that's our naming convention

Reply:
https://github.com/pogosandbox/node-pogo-protos/pull/17#issuecomment-451140914

Reply:
and?

Reply:
I don't see how it's linked

Reply:
https://github.com/Furtif/POGOProtos/blob/master/src/POGOProtos/Data/Telemetry/RpcResponseTime.proto#L7 

because is used my requestype is same original dump

Reply:
still don't see how it's linked.
we are talking here about the naming convention and the name of the message.

about the request type (which is another issue), adding more request type does not modify the telemetry proto, there is no consequence

Reply:
fixed https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/2699be05eaf2516680c0121818fc42aa5a287daf

Reply:
fixed https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/d237965b845295dfd1b3b9705a5580b61d550898